src/components/FormStart/tests.spec.js

```diff
@@ -75,3 +87,29 @@ it('Form start does not start if there are auth errors', () => {
     expect(onFormStart).toHaveBeenCalledTimes(0);
   }
 });
+
+it('Form start page does not show login buttons if an active submission is present', () => {
+  useQuery.mockReturnValue(new URLSearchParams());
+  const onFormStart = jest.fn();
+  const onFormAbort = jest.fn();
+
+  act(() => {
+    render(
+      <IntlProvider locale="nl" messages={messagesNL}>
+        <FormStart
+          form={testForm}
+          onFormStart={onFormStart}
+          onFormAbort={onFormAbort}
+          hasActiveSubmission={true}
+        />
+      </IntlProvider>,
+      container
+    );
+  });
+
+  const buttons = container.getElementsByTagName('button');
+
+  expect(buttons.length).toEqual(2);
+  expect(buttons[0].textContent).toEqual('Continue existing submission');
+  expect(buttons[1].textContent).toEqual('Abort existing submission');
```

Comment:
can you please use testing-library/react patterns for the tests rather than this low-level DOM stuff?

You can check https://github.com/open-formulieren/open-forms-sdk/pull/399 for an example

Additionally, going forward we should focus on writing storybook stories and implementing the `play` function for interaction tests, this automatically takes care of mocking callbacks etc. and is generally a bit more manageable than writing barebones jest tests.

You can find many examples in the https://github.com/open-formulieren/formio-builder repository, e.g. https://github.com/open-formulieren/formio-builder/blob/main/src/components/builder/key.stories.tsx#L96